### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by alderwhiteford**
Minor fixes to documentation, syntax, and structs

### CODE REVIEW

File: backend/src/transactions/club_event.go
Comment (garrettladley) [inline backend/src/transactions/club_event.go:1]: Error message changed and not using limit & offset anymore?

**Review by garrettladley — CHANGES_REQUESTED** [review]
See backend/src/transactions/club_event.go

File: backend/src/transactions/club_event.go
Comment (alderwhiteford) [reply backend/src/transactions/club_event.go:1]: I'll fix

Comment (alderwhiteford) [reply backend/src/transactions/club_event.go:1]: error message is wrong as well lmao. I am tired today

**Review by garrettladley — APPROVED** [review]
LGTM 👏

**Pull request merged**